src/main/java/com/cgdpd/library/mapper/BookMapper.java

```diff
@@ -14,7 +14,6 @@
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
 import org.mapstruct.Named;
-
```

Comment:
This line should be there, use optimise import shortcut `Ctrl + Alt + O` if it stays the same then re open intellij to pickup latest changes from `.editorconfig` file